CONTRIBUTING.md

```diff
@@ -52,8 +52,8 @@ git clone git@github.com:determined-ai/determined.git
 
 - Go (>= 1.18)
 - Python (>= 3.6, <= 3.9)
-- Node (>= 14.16)
-- NPM (>= 6.12)
+- Node (>= 16.14)
```

Comment:
hmm i guess i feel like we should specify 16.14.2 since it's the actual LTS version. >=8.0 sounds good for npm though.

Reply:
decided to go with >=16.13 since that's when 16.x moved into Active LTS and npm was updated to 8 by then. i actually think >=16 would also be fine but i also don't anticipate it being a big problem to require a later version, either, so i'm somewhat arbitrarily landing on 16.13.

Reply:
👍🏼